django_ariadne_jwt/backends.py

```diff
@@ -1,43 +1,163 @@
 """GraphQL auth backends module"""
+import datetime
 from django.contrib.auth import get_user_model
-from .exceptions import JSONWebTokenError
-from .utils import decode_jwt
+from django.conf import settings
+from django.utils import timezone
+from django.utils.module_loading import import_string
+from django.utils.translation import ugettext_lazy as _
+import jwt
+from jwt.exceptions import DecodeError, ExpiredSignatureError
+
+from .exceptions import (
+    AuthenticatedUserRequiredError,
+    ExpiredTokenError,
+    InvalidTokenError,
+    JSONWebTokenError,
+    MaximumTokenLifeReachedError,
+)
+
+
+def load_backend():
+    return import_string(
+        getattr(
+            settings,
+            "JWT_BACKEND",
```

Comment:
Kind of thinking this might should be `DJANGO_ARIADNE_JWT_BACKEND` to avoid any possible confusion.

Reply:
Agreed. I guess we should prefix JWT_EXPIRATION_DELTA and JWT_ALGORITHM as well to keep it consistent.

Reply:
For consistency, I guess let's leave it since everything is using `JWT_` as the prefix. Seems like a candidate for supporting both with a `DeprecationWarning` if the old version is detected.